### PR TITLE
[ICDS Dashboard] awc data changed from daily table to monthly table in 

### PR DESCRIPTION
**Pull request by dhiraj-beehyv**
[Jira Ticket](https://dimagi-dev.atlassian.net/browse/ICDS-747)

Changes for pulling the "AWC launched" from the monthly table instead of the daily table. 

@Rohit25negi, I didn't remove the data being fetched from daily table as it is seems to be used in "Number of AWCs open yesterday" card. Can you please have a look and give some guidance on it.

The tests didn't break so I didn't had to make any changes.

### CODE REVIEW

File: custom/icds_reports/reports/cas_reach_data.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def get_data_for_daily_usage(date, filters):
                     'help_text': awcs_launched_help_text(),
                     'color': None,
                     'percent': None,
-                    'value': get_value(daily_yesterday if current_month_selected else awc_this_month_data, 'awcs'),
+                    'value': get_value(awc_this_month_data, 'awcs'),
```
Comment (Rohit25negi) [inline custom/icds_reports/reports/cas_reach_data.py]: this column is not more needed you should removed: 
https://github.com/dimagi/commcare-hq/pull/25008/files#diff-50b8c0baaef34ca763dda94410624738R42

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
Looks fine. left One change request.

**Comment by Rohit25negi** [comment]
@dhiraj-beehyv Pinging for the change.

**Comment by dhiraj-beehyv** [comment]
@Rohit25negi , I've updated the pr with the changes. Please have a look.

File: custom/icds_reports/reports/cas_reach_data.py [hunk 2]
```diff
@@ -74,7 +73,7 @@ def get_data_for_daily_usage(date, filters):
             'color': get_color_with_green_positive(daily_attendance_percent),
             'percent': daily_attendance_percent,
             'value': get_value(daily_yesterday, 'daily_attendance'),
-            'all': get_value(daily_yesterday, 'awcs'),
+            'all': get_value(awc_this_month_data, 'awcs'),
```
Comment (Rohit25negi) [inline custom/icds_reports/reports/cas_reach_data.py]: @dhiraj-beehyv  can you put these under feature flag. `pre_release_features` variable(boolean) in its caller function should have that info.

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
Can you please put this change under the feature flag

**Comment by dhiraj-beehyv** [comment]
Hi @Rohit25negi ,

Should I put both getting the data from the daily table and the change you requested under the feature flag?

**Review by calellowitz — CHANGES_REQUESTED** [review]
Looks good to me but it looks like rohit left one comment

**Comment by dhiraj-beehyv** [comment]
@calellowitz ,

I've updated the pr with changes which Rohit requested (putting the changes under the feature flag)

**Review by calellowitz — APPROVED** [review]

File: custom/icds_reports/reports/cas_reach_data.py [hunk 3]
```diff
@@ -101,7 +110,7 @@ def get_data_for_daily_usage(date, filters):
                     'help_text': awcs_launched_help_text(),
                     'color': None,
                     'percent': None,
-                    'value': get_value(daily_yesterday if current_month_selected else awc_this_month_data, 'awcs'),
+                    'value': get_value(awc_this_month_data, 'awcs'),
```
Comment (Rohit25negi) [inline custom/icds_reports/reports/cas_reach_data.py]: @dhiraj-beehyv feature flag check will also be applied to this line same as what you did on 86 line.

File: custom/icds_reports/reports/cas_reach_data.py [hunk 4]
```diff
@@ -101,7 +110,7 @@ def get_data_for_daily_usage(date, filters):
                     'help_text': awcs_launched_help_text(),
                     'color': None,
                     'percent': None,
-                    'value': get_value(daily_yesterday if current_month_selected else awc_this_month_data, 'awcs'),
+                    'value': get_value(awc_this_month_data if pre_release_features else daily_yesterday, 'awcs'),
```
Comment (Rohit25negi) [inline custom/icds_reports/reports/cas_reach_data.py]: it should be more like
`daily_yesterday if pre_release_features and current_month_selected else awc_this_month_data`
@dhiraj-beehyv

**Review by Rohit25negi — APPROVED** [review]
Thanks for working on this. Waiting for the build to complete. Will merge after it.

**Pull request merged**